src/utils.ts

```diff
@@ -0,0 +1,5 @@
+export function generateUUIDv4() {
+  return ("" + 1e7 + -1e3 + -4e3 + -8e3 + -1e11).replace(/[018]/g, (c: any) =>
+    (c ^ crypto.getRandomValues(new Uint8Array(1))[0] & 15 >> c / 4).toString(16)
```

Comment:
Nit. Does this need to be cryptographically secure? Just slows you down a bit.

Reply:
It gives it a bit more chance to be unique, and considering this is done once on page initialization it's not much of a burden.